crates/wascc-provider/src/lib.rs

```diff
@@ -105,10 +105,30 @@ impl Provider for WasccProvider {
                 tokio::task::spawn_blocking(move || wascc_run_http(data, env, &pub_key)).await?;
             match http_result {
                 Ok(_) => {
-                    pod.patch_status(client.clone(), &Phase::Running).await;
+                    pod.patch_status(
+                        client.clone(),
+                        Status {
```

Comment:
I think all these `patch_status` calls in this PR would be much easier to read if the status arguments were not inlined and instead moved to a local variable.

Reply:
This better?